components/program/Timeline.vue

```diff
@@ -0,0 +1,354 @@
+            <span class="time text-h4 bold">
+              {{ getHourAndMinuteStringFromString(event.date.start) }}
+            </span>
+          </div>
+
+          <VCard
+            width="600"
+            elevation="0"
+            class="card mb-4"
+            variant="flat"
+            @click="() => navigateTo(`/program/${event.id}`)"
+          >
+            <template #title> {{ event.title }} </template>
+
+            <template v-if="companies[event.companyUID]" #subtitle>
+              {{ companies[event.companyUID].name }}
+            </template>
+
+            <template #text>
+              <span class="card__location mb-2">
+                <VIcon color="primary">mdi-map-marker</VIcon>
+                <NuxtLink
+                  v-if="event.location.map !== 'null'"
+                  :to="event.location.map"
+                  class="link"
+                  @click.stop
+                >
+                  {{ event.location.name }}
+                  <VIcon size="x-small">mdi-open-in-new</VIcon>
+                </NuxtLink>
+                <span v-else>
+                  {{ event.location.name }}
+                </span>
+              </span>
+              <span class="card__timeinterval mb-2">
+                <VIcon color="primary">mdi-clock</VIcon>
+                <span>
+                  {{ getHourAndMinuteStringFromString(event.date.start) }} -
+                  {{ getHourAndMinuteStringFromString(event.date.end) }}
+                </span>
+              </span>
+              <span v-if="event.limitedCapacity" class="card__attendees mb-2">
+                <VIcon
+                  v-if="
+                    Object.hasOwn(event, 'attendants') &&
+                    event?.attendants?.length >= 3
+                  "
+                  color="primary"
+                >
+                  mdi-account-group
+                </VIcon>
+                <VIcon v-else color="primary">mdi-account</VIcon>
+                <span>
+                  <span v-if="Object.hasOwn(event, 'attendants')">
+                    {{ Object.values(event.attendants).length }}
+                  </span>
+                  <span v-else>0</span>
+                  / {{ event.capacity }}
+                </span>
+              </span>
+              <span
+                v-if="!useAuth.isLoggedIn && event.limitedCapacity"
+                class="text-primary"
+              >
+                <br />
+                {{ $t('program.event.sign_in_to_register') }}
+              </span>
+            </template>
+
+            <template
+              v-if="event.limitedCapacity && useAuth.isLoggedIn"
+              #actions
+            >
+              <VBtn
+                v-if="
+                  useAuth.isLoggedIn &&
+                  ((event.attendants &&
+                    !Object.values(event.attendants).includes(
+                      useAuth?.user?.uid
+                    )) ||
+                    !event.attendants)
```

Comment:
Maybe extract this into a computed property.

Reply:
I agree. I sent it out for reapproval too fast. I'm currently refactoring for readability and commenting as well